tests/util.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from git import Actor, Repo
+
+SOME_COMMIT_MESSAGE = "some-commit-message"
+SOME_ACTOR = Actor("some-user", "some-user@wayfair.com")
+SOME_CONTENT = "some-content"
+
+
+@dataclass
+class Repos:
+    local_repo: Repo
+    remote_repo: Repo
```

Comment:
this definition feels very similar to [MultipleTestingRepos](https://github.com/wayfair-incubator/pygitops/blob/main/tests/test_operations.py#L39). is there a way that we can consolidate the definition of `Repos` and `MultipleTestingRepos`?